### DISCUSSION TITLE
星际社区大使飞行项目启动啦！

### DISCUSSION THREAD

**Discussion by maxvint** [filecoin-project/community-china#50] [社区公告]
🛰我们很高兴向大家介绍星际社区大使飞行项目。虽然与其他社区，如IPLD和IPFS社区相比，Filecoin社区还很年轻，但它是非常繁荣、活跃以及获得了热烈地支持。因此，我们很高兴能够为社区成员们创造一个新的机会，让他们继续作为大使来管理Filecoin社区。这个为期6周的飞行项目将专注于使每个社区大使拥有支持积极参与社区的能力，以及发挥关键的社区管理技能，并学习支持Filecoin矿工、用户和开发人员的新方法。

欢迎大家的踊跃参与申请！详情请点击：https://github.com/filecoin-project/community/discussions/116

**Comment by KaixuanZhao** [comment]
IPFS主网最大雅典娜矿池，RRmine global ，一站式采购托管运营，全部数据链上可查，全网算力领先1400PB，需要参与Datacap真实数据存储的朋友可以在小狐狸浏览器中复制打开https://www.rrmglobal.io/#/invite?invite_code=O3DKS2S0 欢迎加入社群 Wechet：kkmmjjnn00

**Comment by KaixuanZhao** [comment]
RRMine，雅典娜矿池，全网算力领先1000PB，需要B端C端方案的滴滴我，交流学习，欢迎加入社群Wechet：kkmmjjnn00

**Comment by zhaobo443** [comment]
FIL联合封装，用户提供质押币，公司提供设备和运维，收益60％,公司扣出20％服务费和20%设备费，合同期1080天，540天质押币退还，t3天封装。可实地考察。V:zhaobo9308

**Comment by RichByDMC** [comment]
挖FIL的伙伴们看过来，一个火爆全网的全新存储项目自带公链钱包NFT等完整生态DMC，唯一能够超过Fil的项目诞生了。主要还能双挖FIL和DMC~~有想法的可以V我：dmctorich
